nvdaHelper/remote/winword.cpp

```diff
@@ -1190,6 +1221,15 @@ void winword_getTextInRange_helper(HWND hwnd, winword_getTextInRange_args* args)
 					break;
 				}
 			}
+			for(auto& i: revisionsVector) {
+				long revStart=get<0>(i);
+				long revEnd=get<1>(i);
+				long revType=get<2>(i);
+				if(!(chunkStartOffset>=revEnd||chunkEndOffset<=revStart)) {

```

Comment:
I somehow find this hard to read. I assume this is the same as:
```suggestion
				if(chunkStartOffset<revEnd&&chunkEndOffset>revStart) {
```